Content.Shared/Projectiles/SharedProjectileSystem.cs

```diff
@@ -1,11 +1,15 @@
 using System.Numerics;
+using System.Reflection.Metadata.Ecma335;
 using Content.Shared.CombatMode.Pacification;
 using Content.Shared.Damage;
+using Content.Shared.Destructible;
 using Content.Shared.DoAfter;
 using Content.Shared.Hands.EntitySystems;
 using Content.Shared.Interaction;
 using Content.Shared.Mobs.Components;
 using Content.Shared.Throwing;
+using Linguini.Syntax.Ast;
+using Microsoft.Extensions.ObjectPool;
```

Comment:
Most of these `using`s are unused, and there are some others that could be cleaned up.